### ISSUE TITLE
Fix build related issues

### ISSUE THREAD

**Issue by ayushmxn** [piskelapp/piskel#1047]
I'll use this issue to track changes made to #1049 to fix build process related errors.

- [x] Allow installation of `node_modules` on Node 14
- [x] Fix test configuration
- [x] Fix desktop builds
- [x] Update dependencies
- [x] Figure out way to reliably [host desktop binaries](https://github.com/piskelapp/piskel/issues/1047#issuecomment-1037302817)

p.s.: @juliandescottes and @bcharbonnier, I sent you guys an email from `ayushmxn@outlook.com`. There is a chance it could have gone into your spam so just wanted to give you a heads up here. Anyways, hopefully chat soon.

**Comment by ayushmxn** [comment]
While trying to build the desktop application, this error shows up. I'm not yet sure if this is the cause though.
```
>> TypeError: os.tmpDir is not a function
Warning: Task "nwjs:windows" not found.
```
I looked into it and this error is resolved in `nw-builder` v3.4.0 but the grunt plugin uses v3.1.1. The code hasn't been updated in 6 years and there is an open [PR](https://github.com/nwjs/grunt-nw-builder/pull/166) to update dependencies. For now I'll try using the fork.

**Comment by ayushmxn** [comment]
I was also thinking about how to distribute desktop builds. According to [this](https://stackoverflow.com/a/68794196) Stack Overflow discussion, we could "publish" the builds on every release. Another way could be to host the executables on the website itself?
